libs/common/src/sql-db-connection/reset-database.sh

```diff
@@ -1,10 +1,26 @@
 #!/bin/bash
```

Comment:
Do you think it's worth having a short explanation anywhere of what this actually does, even if it's a couple of sentences of why we need it in the ledger DB documentation?

Reply:
I'm already planning on updating the ledger docs because they're technically out of date now that we know we can't really add non-nullable columns